### DISCUSSION TITLE
百度网盘无法播放，求助！！

### DISCUSSION THREAD

**Discussion by xq3427** [AlistGo/alist#6577] [Q&A] [answered]
百度网盘出现无法播放，更新至最新版、更换代理模式均无法解决，请问有无大佬知道如何解决

**Comment by SuperDuoDuo** [comment]
同问

**Comment by anwen-anyi** [comment]
这两天经测试都没问题啊，用官方客户端/App试试看能不能下载
前两天似乎不知道发生了什么 有错误码，官方客户端/APP都不能下载改了下密码就好了

**Reply by xq3427** [reply]
我的还是播放不了百度网盘的视频，我试了下阿里云是正常的，已经更新了最新版也不能播放，每次都是重连5次之后播放失败。
但是我openwrt上面搭建的可以正常播放百度网盘的视频。但是电脑上就不行

**Reply by SuperDuoDuo** [reply]
我是搭在香港服务器上 不管本地代理还是web都不行

**Reply by anwen-anyi** [reply]
- 是SVIP账号，网页播放修改UA 可以播放302
- 是SVIP账号，Web代理开启，无需修改UA，搭建机器宽带大可以播放


# 测试过是没问题的

## 302下载
![image](https://github.com/alist-org/alist/assets/56105412/bc3a90f6-989d-4757-a41f-d498570e8030)

## Web代理 在线播放+下载
![image](https://github.com/alist-org/alist/assets/56105412/9ede1230-9013-4f1c-9425-b2e9c344787c)

**Reply by SuperDuoDuo** [reply]
现在要svip了吗 不是不可以了嘛

**Reply by anwen-anyi** [reply, marked as answer]
> 现在要svip了吗 不是不可以了嘛

官方接口一直需要，非官方接口自行测试，不保证100%可用性

**Reply by xq3427** [reply]
感谢解答，已经解决了，换了个另外一个非官方的ua就行了

**Reply by Liceres** [reply]
> * 是SVIP账号，网页播放修改UA 可以播放302
> * 是SVIP账号，Web代理开启，无需修改UA，搭建机器宽带大可以播放
> 
> # 测试过是没问题的
> ## 302下载
> ![image](https://private-user-images.githubusercontent.com/56105412/337653210-bc3a90f6-989d-4757-a41f-d498570e8030.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyODczMzYsIm5iZiI6MTczMjI4NzAzNiwicGF0aCI6Ii81NjEwNTQxMi8zMzc2NTMyMTAtYmMzYTkwZjYtOTg5ZC00NzU3LWE0MWYtZDQ5ODU3MGU4MDMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDE0NTAzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0OWUwM2JjODJhOTdjZjIwM2Q4YmJhMDBlZjQ0NDdiNTgwZGNkZDg2NjM5Njg4MTNiNGFmM2E1NDczM2ViNTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vGrQCUXQt9Bby3-FngNd6uGTq6OLmbIYTzn5eSvyPZc)
> 
> ## Web代理 在线播放+下载
> ![image](https://private-user-images.githubusercontent.com/56105412/337653163-9ede1230-9013-4f1c-9425-b2e9c344787c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyODczMzYsIm5iZiI6MTczMjI4NzAzNiwicGF0aCI6Ii81NjEwNTQxMi8zMzc2NTMxNjMtOWVkZTEyMzAtOTAxMy00ZjFjLTk0MjUtYjJlOWMzNDQ3ODdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDE0NTAzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRlMjE1MTA0ZDZjMjBiN2Q3OTg5MWU1ZDdlYzAyYTY2M2E5N2MxN2I1ZTU4MTFlMjI3ODhlZDhkOTBiZjA3MmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kWZ6NHvfd-6L4RlADx8j26gII-XgFSC01xKDpTb4wd4)

在服务器上用302只能网页改UA在线看吗 不能用webdav用potplayer或者infuse上看吗
